src/app/modules/shared_modules/collectible_details/controller.nim

```diff
@@ -85,6 +85,11 @@ QtObject:
         if self.detailedEntry.updateDataIfSameID(collectible):
           break
 
+  proc processGetCollectibleSocialsResponse(self: Controller, response: JsonNode) =
+    let res = fromJson(response, backend_collectibles.CollectibleSocialsMessage)
+    self.detailedEntry.updateDataIfSameID(res)
+    self.detailedEntryChanged()
```

Comment:
The `updateDataIfSameID` might not do any change, does it make sense to trigger the signal `detailedEntryChanged`?

Reply:
make sense! will remove it as events for dataChanged will be emitted from within the entry as it is